### DISCUSSION TITLE
Duplicated files？

### DISCUSSION THREAD

**Discussion by ucmmesa** [openTCS/opentcs#43] [Q&A] [answered]
Sorry for repeating this question.

How about copy the module files to related directory and then delete the guiceConfig directory?
![opentcs_duplicated_files](https://user-images.githubusercontent.com/11491673/188345246-551eb67b-abc7-4165-8ba6-e811cb04e92d.JPG)

**Comment by swltr** [comment, marked as answer]
`main` contains the actual application source code. `guiceConfig` contains dependency injection configuration that just happens to be maintained in Java files, too. One is application logic, the other one describes how the application's components are put together. Mixing them is not a good idea (in general).
